<launch>
  <!-- Load Parameters (if not already loaded) -->
  <rosparam file="$(find ros_car_cmd)/config/car_params.yaml"/>

  <!-- Startup waypoint_manager node-->
  <node pkg="ros_car_cmd" type="waypoint_manager.py" name="waypoint_manager" output="screen">
    <param name="use_sim_time" value="true"/>
  </node>
</launch>
